utils/constants.ts

```diff
@@ -1,3 +1,16 @@
 export const basicAlphabet = "abcdefghijklmnopqrstuvwxyz0123456789-";
 export const bigAlphabet = "这来";
 export const totalAlphabet = basicAlphabet + bigAlphabet;
+export const UINT_128_MAX = (BigInt(1) << BigInt(128)) - BigInt(1);
+
+export const NFT_CONTRACT_TESTNET = [
+  "0x0154520b48b692bb8b926434bbd24d797e806704af28b6cdcea30ea7db6a996b",
+];
+
+export const NFT_CONTRACT_MAINNET = [
```

Comment:
`PFP_WL_CONTRACTS` is a better naming imo because here we don't know what it is with this naming